docs/admin_guide.md

```diff
@@ -12,7 +12,7 @@ Extension behavior is controlled by __security labels__ and __settings__. Securi
 
 Only superusers can assign security labels. To remove a security label from an object, set it to `NULL`.
 
-The command `SELECT * FROM diffix.show_labels();` displays the current security labels assigned to tables and columns by the extension.
+The command `SELECT * FROM diffix.show_labels();` displays the current security labels assigned to tables and columns by the extension. Note that if this command fails, then probably `pg_diffix` has not been enabled for the database. See the [Administration Tutorial](admin_tutorial.md) for step-by-step instructions.
```

Comment:
If `diffix.show_labels()` fails (the function is not found), it means `pg_diffix` has not been enabled for the current database. If it returns an empty result set, it means no data or users have been configured.
If `diffix.show_settings()` returns nothing, then the extension is not loaded for the current session.